src/main/java/seedu/address/logic/commands/ListAppointmentsCommand.java

```diff
@@ -58,12 +60,36 @@ public CommandResult execute(Model model) {
         LocalDateTime now = LocalDateTime.now();
```

Comment:
Perhaps the name now could be changed to currentDateTime, so that the name is more meaningful?